Automattic/woocommerce-payments
# Rename  'Deposit' to 'Payout' (feature branch)

Fixes https://github.com/Automattic/woocommerce-payments/issues/9595.

#### Changes proposed in this Pull Request
Part of project to rename `deposit` to `payout` -  paJDYF-fwb-p2.

This PR is to merge the feature branch [feature/deposits-payouts-rename](https://github.com/Automattic/woocommerce-payments/tree/feature/deposits-payouts-rename) that consists of renames from 'deposit' to 'payout'.

#### Testing instructions

* All unit tests should pass
* The PR is to merge a feature branch that has all individual PR's renaming specific portions of the WooPayments plugin UI. These changes have already been reviewed before those PR's were merged to this main feature branch. However, it will be good to take a glance to see if there is anything obvious missing / incorrect. 
* Check if the Changelog is appropriate or needs a change. 


-------------------


**Post merge**
